crates/sncast/tests/e2e/invoke.rs

```diff
@@ -3,7 +3,9 @@ use crate::helpers::fixtures::{
     default_cli_args, from_env, get_transaction_hash, get_transaction_receipt,
 };
 use crate::helpers::runner::runner;
+
```

Comment:
were those added by formatter?

Reply:
probably yes, should I remove it?

Reply:
if formatter adds it, I guess removing this will end up with failed CI. Let's make sure that was the case, and if not - remove it. If yes, let's leave this.